app/services/books/search_strategies/s1/factory.rb

```diff
@@ -1,8 +1,8 @@
-module Search::BookVersions::SearchStrategies
+module Books::SearchStrategies::S1
   class Factory
 
     STRATEGY_CLASSES = [
-      S1
+      Strategy
```

Comment:
When this factory moves back up then this because `STRATEGY_CLASSES = [S1::Strategy]`